src/workerd/api/streams/standard.c++

```diff
@@ -3341,7 +3341,8 @@ jsg::Promise<void> WritableStreamJsController::close(jsg::Lock& js, bool markAsH
           js, js.v8TypeError("This WritableStream has been closed."_kj), markAsHandled);
     }
     KJ_CASE_ONEOF(errored, StreamStates::Errored) {
-      return rejectedMaybeHandledPromise<void>(js, errored.getHandle(js), markAsHandled);
```

Comment:
hmm... this *might* be mildly breaking. It might need a compat flag. That's not definite... I can just see the possibility of someone depending on the identity of the error.